packages/appcd-telemetry/src/telemetry.js

```diff
@@ -127,9 +121,9 @@ export default class Telemetry extends Dispatcher {
 		{
 			const { name, version } = osInfo();
 			this.osInfo = {
-				os:       name || 'unknown',
-				osver:    version || 'unknown',
-				platform: process.platform
+				version,
+				name: 		name || process.platform,
+				arch: 		process.arch
```

Comment:
Don't use `process.arch`. Instead, import `arch` from `appcd-util` (we already import `osInfo`) and call `arch()` to determine the correct machine architecture (not the architecture of the compiled Node binary).